mmcls/models/backbones/res2net.py

```diff
@@ -156,9 +159,17 @@ def __init__(self,
                  norm_cfg=dict(type='BN'),
                  scales=4,
                  base_width=26,
+                 drop_path_rate=0.0,
                  **kwargs):
         self.block = block
 
+        if isinstance(drop_path_rate, float):
+            drop_path_rate = np.array([drop_path_rate])
+            drop_path_rate = drop_path_rate.repeat(num_blocks)
```

Comment:
I think `[drop_path_rate] * num_blocks` is enough, we don't need to import numpy and use numpy array here.

Reply:
forgot to update